### PR TITLE
feat: add verbose levels to output with --verbose/-v

### PR DESCRIPTION
**Pull request by katcipis**
Introduce pkg/abstraction for the verboseness. This should enable for some output to be sent on different verboseness levels.

### CODE REVIEW

File: cmd/terramate/cli/cli.go [hunk 1]
```diff
@@ -1120,9 +1130,9 @@ func (c *cli) eval() {
 			fatal(err, "eval %q", exprStr)
 		}
 
-		var out []byte
+		var o []byte
```
Comment (i4ki) [inline cmd/terramate/cli/cli.go:1133]: `i` ok, `j` ok, `s` ok, `v` ok, but `o` ? haha unusual choice but I'm actually fine with it =)

File: cmd/terramate/cli/out/out.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+func New(verboseness int, stdout, stderr io.Writer) O {
+	return O{
+		verboseness: verboseness,
+		stdout:      stdout,
+		stderr:      stderr,
+	}
+}
+
+// Msg send a message to the output with the given verboseness
+func (o O) Msg(verboseness int, format string, args ...interface{}) {
```
Comment (i4ki) [inline cmd/terramate/cli/out/out.go:51]: cute

**Review by i4ki — APPROVED** [review]
lgtm

File: cmd/terramate/cli/cli.go (hunk 1 above)
Comment (katcipis) [reply cmd/terramate/cli/cli.go:1133]: I was in "fixing build errors really fast" mode so I just reduced out to o :laughing:, but can change =P

**Pull request merged**